### ISSUE TITLE
Flatpak Mail doesn't show any messages

### ISSUE THREAD

**Issue by MikuOcta** [elementary/mail#784]
### What Happened?

I installed the mail flatpak and I see no messages are seen. It just shows me a blank space.

![Screenshot from 2022-05-19 20 43 46](https://user-images.githubusercontent.com/81514969/169364123-97f655d8-e030-452a-8ef3-ea5d5788453c.png)
_Mail showing just a blank screen as the message_

NOTE: I do have 2 accounts connected if that matters.

### Steps to Reproduce

1. Remove the normal Mail app
2. Installed the flatpak version from AppCenter (User)
3. See the results

### Expected Behavior

It should show the content of the messages.

### OS Version

6.x (Odin)

### Software Version

Latest release (I have run all updates)

**Comment by pheonixo** [comment]
Can verify that this occurs on ubuntu install, and build by system. This occurs on text from mailing list, yet majority of other emails from list with display. A select all of empty view, and copy, create new view in 'code', then paste will display the contents of the email. I was going to try and debug but vala too strange for me to try and learn in limited time, simple include header, printf does not work.

**Comment by team-cons** [comment]
Hello
I am running the flatpak as well, on Fedora 38, and encountering this issue
-Nothing. No mail. Nowhere. The app loads all mails fine but fails to display anything
-Writing new mail freezes the app
-Reply/ReplyTo/Transfer are greyed out
-The issue persists no matter the provider

Nothing appears when started from commandline
Any idea for a fix please ?

![Bildschirmfoto von 2023-10-16 21 33 56](https://github.com/elementary/mail/assets/147658063/fee1dd3b-3714-4a73-81cc-4a5c9c28d0b7)

**Comment by team-cons** [comment]
No. Doesnt work. Ive tried everything i could think of - Unfortunately other Elementary apps are broken as well, so i will just hold off from the distro for now.

**Comment by Slater91** [comment]
This isn't just with Flatpak, it happens with the native package as well. Currently on elementary 7 with Mail version 7.2.0+r1491+pkg36~ubuntu7.1.

**Comment by donnie-j** [comment]
After an update, mail is now completely unusable due to this issue.  It only occurred infrequently before, now most emails appear blank.  Please bump to higher priority.

Perhaps important here, this is on a high DPI screen, which (at 2x) requires mail to use quite small windows.  
- Testing needs to be done regularly on small screens.

It's certainly a bug with scrolled windows.  In long threads scrolling around will show a few fragments of the email text
- Probably, this is actually a toolkit bug.

**Comment by jeremypw** [comment]
More information about this can be found under #950.  It appears to be an upstream bug between WebView and Gtk.ScrolledWindow. A workaround is to start to reply to the message, which opens it in a separate window where the issue does not occur.  Hopefully this issue will be fixed when later versions of the libraries are used (presumably in elementary 8).  Meanwhile, I started a possible fix/workround in #973.
